app/src/main/java/org/jellyfin/androidtv/data/eventhandling/SocketHandler.kt

```diff
@@ -37,6 +41,7 @@ class SocketHandler(
 	private val dataRefreshService: DataRefreshService,
```

Comment:
There seems to be something off with the alignment of these parameters. :slightly_smiling_face: